UnityServer/functions/src/model/UnityStatus.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2020 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+/* UnityStatus is part of Model layer.
```

Comment:
/* 
 * UnityStatus is part of Model layer.
 * It's an entity that represents the response that the unity game will receive
 */

Reply:
👍